web/src/pages/apd/key-state-personnel/ApdKeyPersonForm.js

```diff
@@ -330,8 +360,53 @@ const PersonForm = forwardRef(
                                 )}
                               </Fragment>
                             </div>
-                            <strong>Total: </strong>
-                            <Dollars>{totalCostFte(year)}</Dollars>
+                            <div className="ds-u-margin-top--2">
+                              <strong>Total: </strong>
+                              <Dollars>{totalCostFte(year)}</Dollars>
+                              <span className="ds-c-field__hint">
+                                Cost with benefits x FTE = Total
+                              </span>
+                            </div>
+                            {apdType === 'MMIS' && (
+                              <Controller
+                                key={`${year}-split`}
+                                name={`split.${year}`}
+                                control={control}
+                                render={({ field: { value, ...props } }) => (
+                                  <ChoiceList
+                                    {...props}
+                                    choices={[
+                                      {
+                                        checked: value?.federal == 90,
+                                        label: '90/10 DDI',
+                                        value: '90-10'
+                                      },
+                                      {
+                                        checked: value?.federal == 75,
+                                        label: '75/25 Operations',
+                                        value: '75-25'
+                                      }
+                                    ]}
+                                    type="radio"
+                                    label="Federal-State Split"
+                                    hint="Select the match rate for Federal Financial Participation applicable to this activity. A FFP of 90-10 means 90% of the total will be Federal government’s share and 10% will be the State’s share."
+                                    // id={`apd-state-profile-pocemail${index}`}
+                                    onChange={e => {
+                                      handleSplitChange(year, e);
+                                    }}
+                                    onBlur={() => {
+                                      trigger(`split.${year}`);
+                                    }}
+                                    errorMessage={
+                                      errors?.split &&
+                                      errors?.split[year] &&
+                                      errors?.split[year]?.state?.message
+                                    }
+                                    errorPlacement="bottom"
+                                  />
+                                )}
+                              />
+                            )}
```

Comment:
**Suggestion:** I think this is chunk of code is a good candidate for breaking it out into a function (usually starting with the word "render", so it's really apparent that all it's doing is returning react code).

So on the first line here you could have something like:
`apdType === 'MMIS' && renderMmisSplitForm()`
and then inside the `renderMmisSplitForm()` function it returns that chunk of react code.

I find that makes the length of the main chunk of react code much shorter and easier to skim through. And you then only need to see the code inside that function if it's relevant to what you're looking at.

Reply:
That's a great idea. I'll update